--- a/libc/inet/resolv.c
+++ b/libc/inet/resolv.c
@@ -2178,7 +2178,7 @@
 	return i;
 }
 libc_hidden_def(gethostbyname_r)
-link_warning(gethostbyname_r, "gethostbyname_r is obsolescent, use getnameinfo() instead.");
+/*link_warning(gethostbyname_r, "gethostbyname_r is obsolescent, use getnameinfo() instead.");*/
 #endif /* L_gethostbyname_r */
 
 
@@ -2496,7 +2496,7 @@
 #undef in6
 }
 libc_hidden_def(gethostbyaddr_r)
-link_warning(gethostbyaddr_r, "gethostbyaddr_r is obsolescent, use getaddrinfo() instead.");
+/*link_warning(gethostbyaddr_r, "gethostbyaddr_r is obsolescent, use getaddrinfo() instead.");*/
 #endif /* L_gethostbyaddr_r */
 
 
@@ -2616,7 +2616,7 @@
 #endif
 }
 libc_hidden_def(gethostbyname)
-link_warning(gethostbyname, "gethostbyname is obsolescent, use getnameinfo() instead.");
+/*link_warning(gethostbyname, "gethostbyname is obsolescent, use getnameinfo() instead.");*/
 #endif /* L_gethostbyname */
 
 
@@ -2638,7 +2638,7 @@
 	return hp;
 }
 libc_hidden_def(gethostbyaddr)
-link_warning(gethostbyaddr, "gethostbyaddr is obsolescent, use getaddrinfo() instead.");
+/*link_warning(gethostbyaddr, "gethostbyaddr is obsolescent, use getaddrinfo() instead.");*/
 #endif /* L_gethostbyaddr */
 
 
